ydb/core/protos/schemeshard/notification_log.proto

```
@@ -0,0 +1,73 @@
+syntax = "proto3";
+
+package NKikimrSchemeShard;
```

Buf (1.65.0)

[error] Files with package "NKikimrSchemeShard" must be within a directory "NKikimrSchemeShard" relative to root but were in directory "ydb/core/protos/schemeshard".

(PACKAGE_DIRECTORY_MATCH)